message RegisterReq
{
	required string username = 1;
	required string password = 2;
}

message RegisterRes
{
	required bool result = 1;
}